lib/TextDocument/Tests/Unit/LocationAssertions.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phpactor\TextDocument\Tests\Unit;
+
+use PHPUnit\Framework\TestCase;
+use Phpactor\TextDocument\Location;
+
+trait LocationAssertions
```

Comment:
anything in the `Tests` folder is private to this package and should not be used by other packages.

for it to be used by other packges it should be in `lib/Test` but:
```
self::assertEquals($location, Location::fromPathAndOffsets('foobar', 1, 2)
```
might be better anyway?

Reply:
Yeah that would have been too obvious for me to spot, of course I'll refactor it.